newrelic/hooks/mlmodel_openai.py

```diff
@@ -403,6 +412,11 @@ async def wrap_chat_completion_async(wrapped, instance, args, kwargs):
 
     completion_id = str(uuid.uuid4())
 
+    # If `.with_streaming_response.` wrapper used, switch to streaming
```

Comment:
For symmetry purposes of the implementation can we move this up into the if not transaction block the same as the async implementation? Right now we are adding the llm true attribute which means it will show up on the AI dashboard as an llm transaction even though we aren't recording any additional info about it which may be confusing to customers.